src/app/drive/services/thumbnail.service.ts

```diff
@@ -164,6 +226,11 @@ export const getThumbnailFrom = async (fileToUpload: FileToUpload): Promise<Thum
     if (firstPDFpageImage) {
       thumbnailFile = await getImageThumbnail(firstPDFpageImage);
     }
+  } else if (thumbnailableVideoExtension.includes(fileType)) {
+    const videoFrame = await getVideoThumbnail(fileToUpload.content);
```

Comment:
Perhaps the name of the function should be changed to `getVideoFrame`, because it returns a video frame and not a thumbnail as such